server/command.go

```diff
@@ -236,3 +237,34 @@ func (p *Plugin) runPopCommand(args []string, extra *model.CommandArgs) (bool, e
 
 	return false, nil
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	todo := model.NewAutocompleteData("todo", "[command]", "Avaliable commands: list, add")
+
+	add := model.NewAutocompleteData("add", "[message]", "Adds a Todo")
+	add.AddTextArgument("Add a todo", "[message]", "")
```

Comment:
Does `Add a todo` make clear what kind of message to enter? Maybe even an example is more clear:
```suggestion
	add.AddTextArgument("E.g. be awesome", "[message]", "")
```